Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM ruby:2.3.3
 
+RUN printf "deb http://archive.debian.org/debian/ jessie main\ndeb-src http://archive.debian.org/debian/ jessie main\ndeb http://security.debian.org jessie/updates main\ndeb-src http://security.debian.org jessie/updates main" > /etc/apt/sources.list
```

Comment:
🤔

Reply:
This line will update the Debian's repositories, and it is required because the command apt-update will fail returning a 404`s response.